pkgs/generate.sh

```diff
@@ -0,0 +1,30 @@
+
+#!/bin/sh
+
+function runInShell {
+  nix-shell -j 4 -p cabal2nix nix-prefetch-scripts coreutils cabal-install stack --run "$*"
+}
+
+set -xe
+set -v
+
+# Get relative path to script directory
+scriptDir=$(dirname -- "$(readlink -f -- "${BASH_SOURCE[0]}")")
+
+source ${scriptDir}/../scripts/set_nixpath.sh
```

Comment:
Might be good to have all paths with vars inside `"quoted"` as we don't know if some people like spaces in their dir names